### PR TITLE
Add Django view example to README

### PR DESCRIPTION
**Pull request by tobiasmcnulty**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -349,6 +349,39 @@ def acme(token):
...
+views.py:
+
+```
+import os
+
+from django.http import HttpResponse, Http404
+
+
+def acme(request, token):
+    def find_key(token):
```
Comment (jezdez) [inline README.md:363]: Could this function be moved outside of the view function to reduce code smell?

File: README.md [hunk 2]
```diff
@@ -349,6 +349,39 @@ def acme(token):
...
+                return os.environ.get("ACME_KEY_{}".format(n))
+    key = find_key(token)
+    if key is None:
+        raise Http404()
+    return HttpResponse(key)
+```
+
+urls.py:
+
+```
```
Comment (jezdez) [inline README.md:378]: Missing import of the acme view function

File: README.md [hunk 3]
```diff
@@ -349,6 +349,39 @@ def acme(token):
...
+
+views.py:
+
+```
+import os
+
+from django.http import HttpResponse, Http404
+
+
+def acme(request, token):
```
Comment (jezdez) [inline README.md]: Maybe call this `acme_challenge` instead?

**Review by jezdez — APPROVED** [review]
Just minor nits, :+1:

**Review by dmathieu — CHANGES_REQUESTED** [review]
Thank you for this PR.
Could you apply the changes suggested by @jezdez?

File: README.md (hunk 1 above)
Comment (dmathieu) [reply README.md:363]: Any reason why you didn't move this function out of the `acme_challenge` one?
I'm not practicing Python a lot. But it looks quite weird.

Also, if you do move it out, could you rename it to `find_acme_key`?

**Comment by ntravis** [comment]
looks exactly like my working implementation. Wish I had seen the PR first, but I came back to add one after I worked out the same code 👍

**Review by dmathieu — APPROVED** [review]

**Pull request merged**